templates/js/m365-message-extension/.vscode/launch.json

```diff
@@ -115,6 +115,18 @@
         }
     ],
     "compounds": [
+        {
+            "name": "Debug in Test Tool (Preview)",
```

Comment:
Add feature flag to control whether it's default F5 option or alternative F5 option?

Reply:
Ok, will add in next PR